### PR TITLE
Engops 3984 pin jnlp/aws docker image to a most recent commit sha

### PR DESCRIPTION
**Pull request by vineethreddy532**
The static analysis tools deployed on the container image gets updated to their latest versions, whenever this docker image is rebuilt. So pinning this docker image to a most recent commit SHA from latest tag.

### CODE REVIEW

**Review by b-edwin-bianchi — APPROVED** [review]
Looks good to me.  Dockerhub build is clean.  But please wait for approval from @saucecon-aanand before merging.  Thank you.

**Review by saucecon-aanand — APPROVED** [review]

**Pull request merged**
